test/wallet-account.test.js

```diff
@@ -106,11 +106,26 @@ it('can request transaction signing', async () => {
     });
 });
 
-it('requests transaction signing automatically when there is no local key', async () => {
-    // TODO: Refactor copy-pasted common setup code
-    let keyPair = nearApi.KeyPair.fromRandom('ed25519');
+function parseTransactionsFromUrl(urlToParse) {
+    const parsedUrl = url.parse(urlToParse, true);
+    expect(parsedUrl).toMatchObject({
+        protocol: 'http:',
+        host: 'example.com',
+        query: {
+            callbackUrl: 'http://example.com/location'
```

Comment:
At some point I think it would be nice to move these strings to constants, so it's more clear  that they're coming from stubbing code